[2024-11-28T11:12:11.532+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: challenge_dag.transform_task manual__2024-11-28T11:12:06.741925+00:00 [queued]>
[2024-11-28T11:12:11.536+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: challenge_dag.transform_task manual__2024-11-28T11:12:06.741925+00:00 [queued]>
[2024-11-28T11:12:11.536+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-11-28T11:12:11.547+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): transform_task> on 2024-11-28 11:12:06.741925+00:00
[2024-11-28T11:12:11.551+0000] {standard_task_runner.py:57} INFO - Started process 48749 to run task
[2024-11-28T11:12:11.553+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'challenge_dag', 'transform_task', 'manual__2024-11-28T11:12:06.741925+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/challenge_dag.py', '--cfg-path', '/tmp/tmp_pp4e9fd']
[2024-11-28T11:12:11.553+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask transform_task
[2024-11-28T11:12:11.577+0000] {task_command.py:410} INFO - Running <TaskInstance: challenge_dag.transform_task manual__2024-11-28T11:12:06.741925+00:00 [running]> on host codespaces-c5d7e2
[2024-11-28T11:12:11.615+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='challenge_dag' AIRFLOW_CTX_TASK_ID='transform_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-28T11:12:06.741925+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-28T11:12:06.741925+00:00'
[2024-11-28T11:12:11.630+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-11-28T11:12:11.633+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=challenge_dag, task_id=transform_task, execution_date=20241128T111206, start_date=20241128T111211, end_date=20241128T111211
[2024-11-28T11:12:11.685+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-11-28T11:12:11.696+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
